modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_claim_establisher.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    class DisabilityCompensationClaimEstablisher < DisabilityCompensationClaimServiceBase
```

Comment:
this is perplexing me a bit. the claim establisher is actually when the claims established, which is more of what the docker does. I think we possibly want to rename this to notify that it is specific to flashes. Also, we won't allow manual submissions of special issues any more. We will set the PACT and only the PACT special issue for the current time, so any special issues in this job should be removed.

Reply:
@FonzMP  I was leaving it until this job to actually save the status on the claim as 'established'. Is that incorrect? Should that be set in the Docker container once we get an evss_id, or somewhere else?

Reply:
Agreed, removed this file, moved the establishment of the claim and flash queue  into the Docker container job